glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Multimedia 
++ #### *Packet* is a formatted unit of data transmitted over network 
++ #### *Frame* is a basic data unit used by media coding formats
++ #### (media)*Track* is equivalent to a single audio or video 
++ #### Web protocols:
+  + #### *UDP*(User Datagram Protocol) is a [transport layer](https://en.wikipedia.org/wiki/OSI_model#Layer_4:_Transport_layer) protocol using connectionless communication
+  + #### *TCP*(Transmission Control Protocol) is a [transport layer](https://en.wikipedia.org/wiki/OSI_model#Layer_4:_Transport_layer) protocol using connection-oriented communication
+  + #### *RTP*(Real-time Transport Protocol) is a network protocol for delivering audio and video over IP networks
+  + #### *HTTP*(Hypertext Transfer Protocol) is a protocol for fetching data from a server by a client
+  + #### (HTTP) *Long Polling* is a technique of keeping an open connection after client's request for as long as new data is not available. This is more efficient than naive repeated polling by a client until new data is received. 
+  + #### *WebRTC*(Web Real-Time Communication) is a free and open-source project providing web browsers and mobile applications with real-time communication (RTC)
+    + #### *signalling* in WebRTC is a process of discovery and establishing connection between two devices
+  + #### *SDP*(Session Description Protocol) is used for describing  multimedia communication sessions for the purposes of announcement and invitation.  
+  + #### *WebSocket* is a communications protocol which enables full-duplex communication between client and server in near real-time 
+  + #### *ICE* is a technique of establishing the most direct connection between two computers, which is used in P2P communication 
+  + #### *STUN*(Session Traversal Utilities for NAT) is a protocol used in interactive communications with hosts hiddent behind a NAT
+  + #### *TURN*(Traversal Using Relays around NAT) is a protocol utilizing TURN server which relays data between clients 
+  + #### *DTLS*(Datagram Transport Layer Security) is a protocol used for providing security to datagram-based applications
++ #### *Container format* is a file format that allows multiple data streams to be embedded into a single file
++ #### *YUV* is a color encoding system which defines one luminance and two chrominance components. By reducing the resolution of chrominance components it is possible to compress an image with miniscule effect on human perception of the image. 
++ #### *Encoding* is a process of converting media from raw format to encoded format (e.g. MP3)
++ #### *Decoding* is a process of converting media from encoded format to raw format
```

Comment:
Perhaps we should say something about "what is the encoded format" and "what is the raw format" and why do we bother with changing these format (it's mostly a compression issue)